Bottom: 18a314b1e085612d6373bd4a91220eb7ff02264a
Top:    ceca5288a6cda21e8ad0bdbba3fcacc8293f261f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 10:31:45 -0500

Add new option for maven configuration

All other repository types had an option to disable configuration.



---
diff --git a/roles/nexus3/tasks/main.yml b/roles/nexus3/tasks/main.yml
index 64d8c88d..1cfa7111 100644
--- a/roles/nexus3/tasks/main.yml
+++ b/roles/nexus3/tasks/main.yml
@@ -74,14 +74,16 @@
   include: nexus-restore.yml
   when: nexus_restore_point is defined
 
-- include: create_repo_maven_proxy_each.yml
-  with_items: "{{ nexus_repos_maven_proxy }}"
+- block:
+    - include: create_repo_maven_proxy_each.yml
+      with_items: "{{ nexus_repos_maven_proxy }}"
 
-- include: create_repo_maven_hosted_each.yml
-  with_items: "{{ nexus_repos_maven_hosted }}"
+    - include: create_repo_maven_hosted_each.yml
+      with_items: "{{ nexus_repos_maven_hosted }}"
 
-- include: create_repo_maven_group_each.yml
-  with_items: "{{ nexus_repos_maven_group }}"
+    - include: create_repo_maven_group_each.yml
+      with_items: "{{ nexus_repos_maven_group }}"
+  when: nexus_config_maven
 
 - block:
     - include: create_repo_docker_hosted_each.yml
